ui/app/routes/variables.js

```diff
@@ -12,7 +14,8 @@ export default class VariablesRoute extends Route.extend(withForbiddenState) {
     }
   }
   model() {
-    // TODO: Populate model from /variables
-    return {};
+    return RSVP.hash({
```

Comment:
`question` Do we want to use `RSVP.hash` here? I think the library is when we want to trigger an asynchronous callback after a promise has resolved or rejected and reference the result.

I think we might be able to get away with:

```
async model() {
 try {
   this.store.findAll('variable');
 } catch(e) {
   // error logic
 }
}
```

I think that the Promise constructor and `try/catch` syntactic sugar were introduced after ES7 which is why we were previously relying on this pattern.